### PR TITLE
metamacros: override previous macros in other versions of this file.

### PR DESCRIPTION
**Pull request by byohay**
Also remove header guard.

### CODE REVIEW

File: extobjc/metamacros.h [hunk 1]
```diff
@@ -6,26 +6,32 @@
  * Released under the MIT license
  */
 
-#ifndef EXTC_METAMACROS_H
-#define EXTC_METAMACROS_H
-
+/// Two things have been done in order to override macros from other versions of this file:
```
Comment (StatusReport) [inline extobjc/metamacros.h]: Put this in the commit description and remove it from the source (I don't think it belongs here).

**Comment by StatusReport** [comment]
Other than that, LGTM.

File: extobjc/metamacros.h (hunk 1 above)
Comment (byohay) [inline extobjc/metamacros.h]: Done.

**Comment by byohay** [comment]
Fixed.

**Pull request merged**